libs/cart/src/drivers/magento/transforms/outputs/cart-payment.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+
+import { DaffCartPaymentMethod } from '../../../../models/cart-payment';
+
+/**
+ * Transforms magento carts into an object usable by daffodil.
+ */
+@Injectable({
+  providedIn: 'root'
+})
+export class DaffMagentoCartPaymentTransformer {
+
+  /**
+   * Transforms the magento CartPayment from the magento cart query into a DaffCartPaymentMethod.
+   * @param response the response from a magento cart query.
+   */
+  transform(responsePayment): DaffCartPaymentMethod {
```

Comment:
Why can't this arg be typed `MagentoCartPaymentMethod`?

Reply:
When testing the spread behavior, `transform` would not accept additional properties.
```
Object literal may only specify known properties, and 'passThrough' does not exist in type 'MagentoCartPaymentMethod'
```
I now see that this is actually due to the way that I was testing it so removing the parameter type wasn't necessary.

Pushing a change that adds the parameter type and fixes the way it is tested.